### PR TITLE
add test for JointTrajectoryClientsContainer

### PR DESCRIPTION
**Pull request by chatblanc-ciel**
- ``fn current position``
  - ok pattern
  - err pattern

- ``fn send position``
  - ok pattern
  - err pattern

Remaking pull request.

### CODE REVIEW

File: arci/src/clients/joint_trajectory_clients_container.rs [hunk 1]
```diff
@@ -207,4 +207,211 @@ mod tests {
...
+        let container = JointTrajectoryClientsContainer::new(clients);
+        let current = container.current_joint_positions();
+        let expect = vec![1.0_f64, 3.0_f64, 2.2_f64, 4.0_f64];
+
+        let positions = current.unwrap();
+
+        positions
+            .iter()
+            .zip(expect)
+            .for_each(|(pos, correct)| assert_approx_eq!(*pos, correct));
```
Comment (OTL) [inline arci/src/clients/joint_trajectory_clients_container.rs:318]: I think it's better to check current is error, I mean

```rust
assert!(current.is_err());
```

and remove L315-318, L265.
What do you think?

File: arci/src/clients/joint_trajectory_clients_container.rs [hunk 2]
```diff
@@ -207,4 +207,211 @@ mod tests {
...
+            Dummy {
+                name: vec![String::from("part2"), String::from("low")],
+                pos: vec![2.2_f64, 4.0_f64],
+            },
+        ];
+
+        let container = JointTrajectoryClientsContainer::new(clients);
+        let pos = vec![3.4_f64, 5.8_f64, 0.1_f64, 2.5_f64];
+        let duration = std::time::Duration::from_secs(5);
+        let _ = container.send_joint_positions(pos, duration).unwrap();
```
Comment (OTL) [inline arci/src/clients/joint_trajectory_clients_container.rs:365]: I want to use `await` to check wait logic is working.

```suggestion
        container.send_joint_positions(pos, duration).unwrap().await;
```

To do that, we need `#[tokio::test]` and `async` to the function name.
You can see the examples [here](https://github.com/openrr/openrr/blob/main/arci/tests/test_waits.rs)

File: arci/src/clients/joint_trajectory_clients_container.rs (hunk 1 above)
Comment (chatblanc-ciel) [reply arci/src/clients/joint_trajectory_clients_container.rs:318]: I agree with your purpose.
I'll change it.

In other view, ``unwrap`` is often using case.
So, is ``unwrap`` test necessary?

File: arci/src/clients/joint_trajectory_clients_container.rs (hunk 2 above)
Comment (chatblanc-ciel) [reply arci/src/clients/joint_trajectory_clients_container.rs:365]: Is ` assert!(container.send_joint_positions(pos, duration).is_err());` bat test?

I want to know the difference between `unwrap test` and ` unwrap + await`.

**Review by OTL — APPROVED** [review]

**Review by OTL — APPROVED** [review]

**Pull request merged**